documentation/book/cluster-operator.adoc

```diff
@@ -78,6 +78,10 @@ A JSON string representing the storage configuration for the Kafka broker nodes.
 `kafka-metrics-config`::
 A JSON string representing the JMX exporter configuration for exposing metrics from Kafka broker nodes.
 When this field is absent no metrics will be exposed.
+`kafka-log-config`::
```

Comment:
We are not really explaining here what the fields are going to be. I think we might want to list the loggers ... at least in the example below if nothing else.